builds/azure-pipelines/build-release-java.yml

```diff
@@ -0,0 +1,30 @@
+steps:
+  - task: Maven@3
+    displayName: Build library
+    inputs:
+      mavenPomFile: java-library/pom.xml
+      options: --batch-mode --define=gpg.skip --define=org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --update-snapshots
+
+  - powershell: |
+      $prefix = 'azure-functions-java-library-sql-0.1.0'
+      $source = 'java-library'
+      $destination = '$(Build.ArtifactStagingDirectory)/0.1.0'
+      New-Item $destination -ItemType Directory
+      Copy-Item "$source/pom.xml" "$destination/$prefix.pom"
+      Copy-Item "$source/target/$prefix.jar" "$destination/$prefix.jar"
+      Copy-Item "$source/target/$prefix-javadoc.jar" "$destination/$prefix-javadoc.jar"
+      Copy-Item "$source/target/$prefix-sources.jar" "$destination/$prefix-sources.jar"
+    displayName: Copy output files
+
+  - task: ManifestGeneratorTask@0
+    displayName: Generate SBOM manifest
+    inputs:
+      buildDropPath: $(Build.ArtifactStagingDirectory)/0.1.0
+      packageName: Azure Functions SQL Java Bindings
+      packageVersion: '0.1.0'
```

Comment:
Do we actually need to specify the version? I don't see that in ADS

Reply:
I think it's not required - I'll remove it